graphql2/gqlgen.yml

```diff
@@ -107,3 +107,5 @@ models:
       - github.com/target/goalert/schedule.RuleID
   MessageLogConnectionStats:
     model: github.com/target/goalert/notification.SearchOptions
+  MessageLogSegmentBy:
+    model: github.com/target/goalert/graphql2.MessageLogSegmenyBy
```

Comment:
Can we just let gqlgen generate this? It already handles enums natively.

Reply:
I needed to convert it before without it erroring out, but I'll remove this and regenerate and see what happens.